automates/text_reading/src/main/scala/org/clulab/aske/automates/alignment/Aligner.scala

```diff
@@ -62,14 +62,15 @@ class PairwiseW2VAligner(val w2v: Word2Vec, val relevantArgs: Set[String]) exten
   def this(w2vPath: String, relevantArgs: Set[String]) = this(new Word2Vec(w2vPath), relevantArgs)
 
   def alignMentions(srcMentions: Seq[Mention], dstMentions: Seq[Mention]): Seq[Alignment] = {
-    alignTexts(srcMentions.map(Aligner.getRelevantText(_, relevantArgs)), dstMentions.map(Aligner.getRelevantText(_, relevantArgs)))
+    alignTexts(srcMentions.map(Aligner.getRelevantText(_, relevantArgs).toLowerCase()), dstMentions.map(Aligner.getRelevantText(_, relevantArgs).toLowerCase()))
   }
 
   def alignTexts(srcTexts: Seq[String], dstTexts: Seq[String]): Seq[Alignment] = {
     val exhaustiveScores = for {
       (src, i) <- srcTexts.zipWithIndex
       (dst, j) <- dstTexts.zipWithIndex
       score = compare(src, dst) + 2 * (1.0 / (editDistance(src, dst) + 1.0))
+//      score =  (1.0 / (editDistance(src, dst) + 1.0))
```

Comment:
@maxaalexeeva what is the purpose of adding a commented out version of `score` in this method?

Reply:
@pauldhein, I think that was the original score we used/ the score without using word embeddings. I'd like to keep it, but I will make a note.